conf.d/dns_check.yaml.example

```diff
@@ -9,3 +9,7 @@ instances:
       # Specify an (optional) `record_type` to customize the record type
       # queried by the check (default: "A")
       # record_type: A
+
+    - hostname: nxdomain.example.org
```

Comment:
Could you add here a short explanation of why/how one would use an `NXDOMAIN` `record_type`, since its behavior is quite different from regular `record_type`s? (a summarized version of your PR comment would be perfect)